packages/connect-core/src/connections/ConnectorInterface.ts

```diff
@@ -11,7 +11,9 @@ import Role from '../entities/Role'
 export interface ConnectorInterface {
   chainId?: number
   permissionsForOrg(orgAddress: string): Promise<Permission[]>
```

Comment:
Following are the new functions that core level connectors need to implement. Notice how:
1. It uses separate functions for subscriptions
2. Some single-call interface functions do not have a subscription analogy. Thought it doesn't make sense in some cases.